frontend/src/pages/chat/components/chat-layout/chat-layout.tsx

```diff
@@ -68,7 +68,8 @@ const ChatLayout: React.FC<Properties> = ({ filter }) => {
     if (createMessageDataStatus === DataStatus.FULFILLED) {
       void dispatch(chatActions.getAllChats(filter));
     }
-  }, [createMessageDataStatus, dispatch, filter]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Not completely sure that is needed

Reply:
@v0ldemar01 i deleted filter from dependencies to avoid 2 request to backend, because filter in dependencies is useless in this case. but without commenting eslint rule i can`t do that

Reply:
don't we need to getAllMessages on filter change? or how does this work